### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

- Update styling & add CENNZnet wallet

#### Adds
 - SignIn popup
 - `useLocalStorage` hook

#### Changes

 - add `DiscordHandler` and `TwitterHandler` buttons
 - allow user to connect CENNZ wallet and select account
 - add details about the Identity module

#### Preview

https://app-identity-git-update-styling-cennznet.vercel.app/

### CODE REVIEW

**Review by ken-futureverse — CHANGES_REQUESTED** [review]
Code looks good, Interface looks good, one thing I would change is the Discord and Twitter buttons should not appear until you connect to CENNZnet, not the other way around

**Comment by aidan-starke** [comment]
> Code looks good, Interface looks good, one thing I would change is the Discord and Twitter buttons should not appear until you connect to CENNZnet, not the other way around

As the auth process involves a redirect, the CENNZnet session would be wiped each time they come back

File: packages/next-app/libs/hooks/useLocalStorage.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		}
+		try {
+			const item = window.localStorage.getItem(key);
+			return item ? JSON.parse(item) : initialValue;
+		} catch (error) {
+			console.log(error);
+			return initialValue;
+		}
+	});
+	const setValue = (value: T | ((val: T) => T)) => {
```
Comment (ken-futureverse) [inline packages/next-app/libs/hooks/useLocalStorage.ts]: nice custom hook! i wonder if it's better to use `useEffect` to handle the initial `setStoredValue` part. As I can see here you have to do a bit of guarding due to server side rendering.

Server side rendering will never touch `useEffect`, so you can safely use `window` object in there.

**Review by ken-futureverse — APPROVED** [review]
One more thing from me is, do we want to allow user to pick however they want to connect, or would it be simpler to force them to connect the CENNZnet first, then provide them options between Twitter / Discord after.

**Comment by aidan-starke** [comment]
> One more thing from me is, do we want to allow user to pick however they want to connect, or would it be simpler to force them to connect the CENNZnet first, then provide them options between Twitter / Discord after.

I'm happy with the way it is now, but if we want to we could lose the twitter/discord buttons, and once they have connected CENNZnet they can use the same button to popup a sign in page. from that sign in page they could select Twitter / Discord.

I am having an issue with the sign-in popup though, looking in to it now
Screen Shot 2022-03-22 at 1 47 57 PM

**Pull request merged**